templates/part.content.php

```diff
@@ -21,6 +21,10 @@
 ?>
 <div id="surveyResults">
 
+	<?php if(empty($_['statistics'])) { ?>
+		No statistics available, please try later
```

Comment:
@LukasReschke the whole page is not translated, look at the rest of the template file. So yes, we should add translations but no, not for now :wink: